packages/back-end/src/models/InformationSchemaTablesModel.ts

```diff
@@ -0,0 +1,58 @@
+import omit from "lodash/omit";
+import mongoose from "mongoose";
+import uniqid from "uniqid";
+import { Column } from "../types/Integration";
+
+const informationSchemaTablesSchema = new mongoose.Schema({
+  id: String,
+  organization: String,
+  table_name: String,
+  columns: [
+    {
+      id: String,
+      column_name: String,
+      data_type: String,
+      path: String,
+    },
```

Comment:
Same feedback as above re: nested schema validation.

Reply:
Updated with the validator.